src/selectors/per-thread/markers.js

```diff
@@ -360,25 +360,50 @@ export function getMarkerSelectorsPerThread(
   );
 
   /**
-   * This filters out network markers from the list of all markers, so that
-   * they'll be displayed in the marker chart.
+   * The marker chart ignores the preview selection, as it handles its own zooming
+   * behavior. It uses the SearchFilteredMarkerIndexes instead. It shows markers
```

Comment:
optional nit: what do you think about adding that we handle the zooming behavior in the marker chart "for performance"?